stable/concourse/Chart.yaml

```diff
@@ -9,15 +9,11 @@ keywords:
 - concourse.ci
 home: https://concourse-ci.org/
 sources:
-- https://github.com/concourse/bin
-- https://github.com/kubernetes/charts
+- https://github.com/concourse/concourse
+- https://github.com/helm/charts
 maintainers:
-- name: frodenas
-  email: frodenas@gmail.com
-- name: viglesiasce
```

Comment:
Quick note for transparency - removing these two lines were not intentional. Discussing internally about whether to make a follow-up PR to add back.